.gitignore

```diff
@@ -348,3 +348,4 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+/ArtGallery.Services.Api/appsettings.Development.json
```

Comment:
As we are ignoring the 'Development.json' please delete the file from this commit.

Reply:
deleted.